backend_plugin/src/main/evaluators/Evaluator.java

```diff
@@ -54,11 +55,11 @@ public Evaluator(EvaluatorManager em, ITextEditor textEditor) {
 	 * @param textEditor The text document editor this Evaluator is evaluating
 	 */
 	private void initializeFeatureEvaluators(ITextEditor textEditor) {
-		this.featureEvaluators.add(new BlockCommentEvaluator(this.document));
-		this.featureEvaluators.add(new RemoveImportEvaluator(textEditor));
-		this.featureEvaluators.add(new AddImportEvaluator(this.document));
-		this.featureEvaluators.add(new CorrectIndentationEvaluator(this.document));
-		this.featureEvaluators.add(new TrailingWhiteSpaceEvaluator(this.document));
+		this.featureEvaluators.add(new BlockCommentEvaluator(FeatureSuggestion.BLOCK_COMMENT_EVAL, this.document));
+		this.featureEvaluators.add(new RemoveImportEvaluator(FeatureSuggestion.REMOVE_IMPORT_EVAL, textEditor));
+		this.featureEvaluators.add(new AddImportEvaluator(FeatureSuggestion.ADD_IMPORT_EVAL, this.document));
+		this.featureEvaluators.add(new CorrectIndentationEvaluator(FeatureSuggestion.CORRECT_INDENTATION_EVAL, this.document));
+		this.featureEvaluators.add(new TrailingWhiteSpaceEvaluator(FeatureSuggestion.TRAILING_WHITE_SPACE_EVAL, this.document));
```

Comment:
Rather than having to pass the String finals all the way down the chain into each evaluation function, we could access them directly from the evaluation functions since they are public static. That would make things a lot cleaner and reduce the amount of information we have to pass between our various classes

Reply:
Good point. There's no reason to have this weird part where values are passed when the evaluators can grab the strings directly themselves upon creation.